### ISSUE TITLE
Errors found in execution tests from release beta-2 on Linux

### ISSUE THREAD

**Issue by avancinirodrigo** [TerraME/terrame#1213]
```bash
Creating temporary directory
Testing installed packages
Cleaning packages
Cleaning 'build'
Cleaning 'buildafile'
Cleaning 'buildunnecfiles'
Cleaning 'buildunneclean'
Cleaning 'depend'
Cleaning 'depend2'
Cleaning 'images'
Cleaning 'load-forgotten'
Cleaning 'load-twice'
Cleaning 'load-wrong'
Cleaning 'models'
Cleaning 'nodata'
Cleaning 'nodatadotlua'
Cleaning 'nodescription'
Cleaning 'nodoc'
Cleaning 'noexamples'
Cleaning 'nofont'
Cleaning 'nofontdotlua'
Cleaning 'noload'
Cleaning 'nolua'
Cleaning 'onerror'
Cleaning 'onerrorbuild'
Cleaning 'tabular'
Cleaning 'twoerrors'
Cleaning 'twoerrorsbuild'
Cleaning 'withoutdatafont'
Cleaning 'wrongdata'
Cleaning 'wrongfont'
Copying packages
Copying 'build'
Copying 'buildafile'
Copying 'buildunnecfiles'
Copying 'buildunneclean'
Copying 'depend'
Copying 'depend2'
Copying 'images'
Copying 'load-forgotten'
Copying 'load-twice'
Copying 'load-wrong'
Copying 'models'
Copying 'nodata'
Copying 'nodatadotlua'
Copying 'nodescription'
Copying 'nodoc'
Copying 'noexamples'
Copying 'nofont'
Copying 'nofontdotlua'
Copying 'noload'
Copying 'nolua'
Copying 'onerror'
Copying 'onerrorbuild'
Copying 'tabular'
Copying 'twoerrors'
Copying 'twoerrorsbuild'
Copying 'withoutdatafont'
Copying 'wrongdata'
Copying 'wrongfont'
Removing files
Removing '/home/developer/terrame/bin/packages/onerror/log/mac/chart_cell.bmp'
Removing '/home/developer/terrame/bin/packages/twoerrors/log/mac/discrete-rain.log'
Removing '/home/developer/terrame/bin/packages/twoerrors/log/mac/chart_cell_select.bmp'
Removing '/home/developer/terrame/bin/packages/nodatadotlua/data.lua'
Removing '/home/developer/terrame/bin/packages/nofontdotlua/font.lua'
Removing '/home/developer/terrame/bin/packages/models/tests'
Testing group 'basic'
Testing basictrace
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing builderror1
Error: Strings do not match (line 1):
Log file: 'Building package 'abcdef'
'.
Test:     'Building package 'abcdef''.
Testing builderror2
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing builderror3
Error: Strings do not match (line 1):
Log file: 'Option not recognized: -clea
'.
Test:     'Option not recognized: -clea'.
Testing builderror4
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing builderror5
Error: Strings do not match (line 1):
Log file: 'Building package 'onerror'
'.
Test:     'Building package 'onerror''.
Testing depend
Error: Strings do not match (line 1):
Log file: 'Building package 'depend'
'.
Test:     'Building package 'depend''.
Testing depend2
Error: Strings do not match (line 1):
Log file: 'Building package 'depend2'
'.
Test:     'Building package 'depend2''.
Testing doc
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'abcdef'
'.
Test:     'Building documentation for package 'abcdef''.
Testing example
Error: Strings do not match (line 1):
Log file: '	TFT	AD	COOP1	PAVLOV	RANDOM	TF2T	NTFT	_SUM_
'.
Test:     '	TFT	AD	COOP1	PAVLOV	RANDOM	TF2T	NTFT	_SUM_'.
Testing fulltrace
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing help
Error: Strings do not match (line 1):
Log file: '
'.
Test:     ''.
Testing noconfigure
Error: Strings do not match (line 1):
Log file: 'Package "base" does not have any Model.
'.
Test:     'Package "base" does not have any Model.'.
Testing nodoc
Error: Strings do not match (line 1):
Log file: 'It was not possible to find the documentation of package 'nodoc'.
'.
Test:     'It was not possible to find the documentation of package 'nodoc'.'.
Testing noexample
Error: Strings do not match (line 2):
Log file: 'Please use one from the list below:
'.
Test:     'Please use one from the list below:'.
Testing qwerty4321
Testing test
Error: Strings do not match (line 1):
Log file: 'Loading package 'abcdef'
'.
Test:     'Loading package 'abcdef''.
Testing tmp
Testing traceagent
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing tracecell
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing tracecellpair
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing traceconnection
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing traceelement
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing tracefile
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing traceneighbor
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing traceneighborhood
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing traceorderedelement
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing tracepackage
Testing tracesocialnetwork
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing tracesyntax
Testing uninstall
Error: Strings do not match (line 1):
Log file: 'Package 'abcdef' is not installed.
'.
Test:     'Package 'abcdef' is not installed.'.
Testing version
Error: Strings do not match (line 1):
Log file: 'TerraME - Terra Modeling Environment
'.
Test:     'TerraME - Terra Modeling Environment'.
Testing group 'build'
Testing build
Command 'terrame -package build -build config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing buildafile
Command 'terrame -package buildafile -build' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildafile'
'.
Test:     'Building package 'buildafile''.
Testing buildunnecfiles
Command 'terrame -package buildunnecfiles -build config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildunnecfiles'
'.
Test:     'Building package 'buildunnecfiles''.
Testing buildunneclean
Command 'terrame -package buildunneclean -build config/all.lua -clean' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildunneclean'
'.
Test:     'Building package 'buildunneclean''.
Testing noexamples
Command 'terrame -package noexamples -build' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'noexamples'
'.
Test:     'Building package 'noexamples''.
Testing onerrorbuild
Command 'terrame -package onerrorbuild -build config/all.lua -clean' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'onerrorbuild'
'.
Test:     'Building package 'onerrorbuild''.
Testing twoerrorsbuild
Command 'terrame -package twoerrorsbuild -build config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'twoerrorsbuild'
'.
Test:     'Building package 'twoerrorsbuild''.
Testing group 'doc'
Testing doc
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'base'
'.
Test:     'Building documentation for package 'base''.
Testing images
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'images'
'.
Test:     'Building documentation for package 'images''.
Testing nodata
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodata'
'.
Test:     'Building documentation for package 'nodata''.
Testing nodatadotlua
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodatadotlua'
'.
Test:     'Building documentation for package 'nodatadotlua''.
Testing nodescription
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodescription'
'.
Test:     'Building documentation for package 'nodescription''.
Testing nofont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nofont'
'.
Test:     'Building documentation for package 'nofont''.
Testing nofontdotlua
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nofontdotlua'
'.
Test:     'Building documentation for package 'nofontdotlua''.
Testing noload
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'noload'
'.
Test:     'Building documentation for package 'noload''.
Testing onerror
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'onerror'
'.
Test:     'Building documentation for package 'onerror''.
Testing tabular
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'tabular'
'.
Test:     'Building documentation for package 'tabular''.
Testing twoerrors
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'twoerrors'
'.
Test:     'Building documentation for package 'twoerrors''.
Testing withoutdatafont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'withoutdatafont'
'.
Test:     'Building documentation for package 'withoutdatafont''.
Testing wrongdata
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'wrongdata'
'.
Test:     'Building documentation for package 'wrongdata''.
Testing wrongfont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'wrongfont'
'.
Test:     'Building documentation for package 'wrongfont''.
Testing group 'mode'
Testing debug
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing normal
Error: Strings do not match (line 1):
Log file: 'nil
'.
Test:     'nil'.
Testing quiet
Error: Strings do not match (line 1):
Log file: 'nil
'.
Test:     'nil'.
Testing strict
Error: Strings do not match (line 1):
Log file: 'scripts/basic.lua:1: Warning: Variable 'abc' is not declared.
'.
Test:     'scripts/basic.lua:1: Warning: Variable 'abc' is not declared.'.
Testing group 'observer'
Testing chart
Testing clock
Testing map
Testing observer
Error: Strings do not match (line 2):
Log file: 'Stack traceback:
'.
Test:     'Stack traceback:'.
Testing textscreen
Testing visualtable
Testing group 'package'
Testing example
Error: Strings do not match (line 1):
Log file: '0	1.98	0.7762392	1.9602
'.
Test:     '0	1.98	0.7762392	1.9602'.
Testing loadforgotten
Testing loadtwice
Testing loadwrong
Testing nodescription
Testing noexamples
Error: Strings do not match (line 1):
Log file: 'Package 'noexamples'
'.
Test:     'Package 'noexamples''.
Testing noexamplesexample
Error: Strings do not match (line 1):
Log file: 'Package 'noexamples' has no examples
'.
Test:     'Package 'noexamples' has no examples'.
Testing nolua
Testing nopackage
Error: Strings do not match (line 1):
Log file: 'A package should be specified after -package.
'.
Test:     'A package should be specified after -package.'.
Testing group 'sketch'
Testing base
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing data
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing font
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing terralib
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing test
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing void
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing group 'test'
Testing noexamples
Error: Strings do not match (line 1):
Log file: 'Loading package 'noexamples'
'.
Test:     'Loading package 'noexamples''.
Testing noload
Error: Strings do not match (line 1):
Log file: 'Loading package 'noload'
'.
Test:     'Loading package 'noload''.
Testing nolog
Error: Strings do not match (line 3):
Log file: <end of file>
Test: 'Stack traceback:'.
Testing onedirectory
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Testing onefile
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Testing onerror
Error: Strings do not match (line 3):
Log file: 'Loading package 'onerror'
'.
Test:     'Loading package 'onerror''.
Testing onetest
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Testing pattern
Error: Strings do not match (line 2):
Log file: 'Loading package 'base'
'.
Test:     'Loading package 'base''.
Testing twodirectories
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Testing twoerrors
Error: Strings do not match (line 3):
Log file: 'Loading package 'twoerrors'
'.
Test:     'Loading package 'twoerrors''.
Testing twofiles
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Testing twotest
Error: Strings do not match (line 3):
Log file: 'Loading package 'base'
'.
Test:     'Stack traceback:'.
Checking builds
Checking buildunneclean_0.1.zip
Checking buildafile_0.1.zip
Checking noexamples_0.1.zip
Checking twoerrorsbuild_0.1.zip
Checking buildunnecfiles_0.1.zip
Checking build_0.1.zip
Checking onerrorbuild_0.1.zip
Removing packages
Package 'build' was successfully uninstalled.
Package 'buildafile' was successfully uninstalled.
Package 'buildunnecfiles' was successfully uninstalled.
Package 'buildunneclean' was successfully uninstalled.
Package 'depend' was successfully uninstalled.
Package 'depend2' was successfully uninstalled.
Package 'images' was successfully uninstalled.
Package 'load-forgotten' was successfully uninstalled.
Package 'load-twice' was successfully uninstalled.
Package 'load-wrong' was successfully uninstalled.
Package 'models' was successfully uninstalled.
Package 'nodata' was successfully uninstalled.
Package 'nodatadotlua' was successfully uninstalled.
Package 'nodescription' was successfully uninstalled.
Package 'nodoc' was successfully uninstalled.
Package 'noexamples' was successfully uninstalled.
Package 'nofont' was successfully uninstalled.
Package 'nofontdotlua' was successfully uninstalled.
Package 'noload' was successfully uninstalled.
Package 'nolua' was successfully uninstalled.
Package 'onerror' was successfully uninstalled.
Package 'onerrorbuild' was successfully uninstalled.
Package 'tabular' was successfully uninstalled.
Package 'twoerrors' was successfully uninstalled.
Package 'twoerrorsbuild' was successfully uninstalled.
Package 'withoutdatafont' was successfully uninstalled.
Package 'wrongdata' was successfully uninstalled.
Package 'wrongfont' was successfully uninstalled.
Testing from local directories
Removing files
Removing 'onerror/log/mac/chart_cell.bmp'
Removing 'twoerrors/log/mac/discrete-rain.log'
Removing 'twoerrors/log/mac/chart_cell_select.bmp'
Removing 'nodatadotlua/data.lua'
Removing 'nofontdotlua/font.lua'
Removing 'models/tests'
Testing group 'basic'
Testing builderror2
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing builderror3
Error: Strings do not match (line 1):
Log file: 'Option not recognized: -clea
'.
Test:     'Option not recognized: -clea'.
Testing builderror4
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing builderror5
Error: Strings do not match (line 1):
Log file: 'Building package 'onerror'
'.
Test:     'Building package 'onerror''.
Testing depend
Error: Strings do not match (line 1):
Log file: 'Building package 'depend'
'.
Test:     'Building package 'depend''.
Testing depend2
Error: Strings do not match (line 1):
Log file: 'Building package 'depend2'
'.
Test:     'Building package 'depend2''.
Testing nodoc
Error: Strings do not match (line 1):
Log file: 'It was not possible to find the documentation of package 'nodoc'.
'.
Test:     'It was not possible to find the documentation of package 'nodoc'.'.
Testing qwerty4321
Testing group 'build'
Testing build
Command 'terrame -package build -build ../config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'build'
'.
Test:     'Building package 'build''.
Testing buildafile
Command 'terrame -package buildafile -build' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildafile'
'.
Test:     'Building package 'buildafile''.
Testing buildunnecfiles
Command 'terrame -package buildunnecfiles -build ../config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildunnecfiles'
'.
Test:     'Building package 'buildunnecfiles''.
Testing buildunneclean
Command 'terrame -package buildunneclean -build ../config/all.lua -clean' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'buildunneclean'
'.
Test:     'Building package 'buildunneclean''.
Testing noexamples
Command 'terrame -package noexamples -build' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'noexamples'
'.
Test:     'Building package 'noexamples''.
Testing onerrorbuild
Command 'terrame -package onerrorbuild -build ../config/all.lua -clean' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'onerrorbuild'
'.
Test:     'Building package 'onerrorbuild''.
Testing twoerrorsbuild
Command 'terrame -package twoerrorsbuild -build ../config/all.lua' stopped with an error.
PANIC: unprotected error in call to Lua API (/home/developer/terrame/bin/lua/build.lua:313: attempt to concatenate a nil value (field '?'))
Error: Strings do not match (line 1):
Log file: 'Building package 'twoerrorsbuild'
'.
Test:     'Building package 'twoerrorsbuild''.
Testing group 'doc'
Testing images
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'images'
'.
Test:     'Building documentation for package 'images''.
Testing nodata
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodata'
'.
Test:     'Building documentation for package 'nodata''.
Testing nodatadotlua
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodatadotlua'
'.
Test:     'Building documentation for package 'nodatadotlua''.
Testing nodescription
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nodescription'
'.
Test:     'Building documentation for package 'nodescription''.
Testing nofont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nofont'
'.
Test:     'Building documentation for package 'nofont''.
Testing nofontdotlua
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'nofontdotlua'
'.
Test:     'Building documentation for package 'nofontdotlua''.
Testing noload
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'noload'
'.
Test:     'Building documentation for package 'noload''.
Testing onerror
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'onerror'
'.
Test:     'Building documentation for package 'onerror''.
Testing tabular
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'tabular'
'.
Test:     'Building documentation for package 'tabular''.
Testing twoerrors
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'twoerrors'
'.
Test:     'Building documentation for package 'twoerrors''.
Testing withoutdatafont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'withoutdatafont'
'.
Test:     'Building documentation for package 'withoutdatafont''.
Testing wrongdata
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'wrongdata'
'.
Test:     'Building documentation for package 'wrongdata''.
Testing wrongfont
Error: Strings do not match (line 1):
Log file: 'Building documentation for package 'wrongfont'
'.
Test:     'Building documentation for package 'wrongfont''.
Testing group 'mode'
Testing group 'observer'
Testing group 'package'
Testing example
Error: Strings do not match (line 1):
Log file: '0	1.98	0.7762392	1.9602
'.
Test:     '0	1.98	0.7762392	1.9602'.
Testing loadforgotten
Testing loadtwice
Testing loadwrong
Testing nodescription
Testing noexamples
Error: Strings do not match (line 1):
Log file: 'Package 'noexamples'
'.
Test:     'Package 'noexamples''.
Testing noexamplesexample
Error: Strings do not match (line 1):
Log file: 'Package 'noexamples' has no examples
'.
Test:     'Package 'noexamples' has no examples'.
Testing nolua
Testing nopackage
Error: Strings do not match (line 1):
Log file: 'A package should be specified after -package.
'.
Test:     'A package should be specified after -package.'.
Testing group 'sketch'
Testing data
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing font
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing terralib
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing test
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing void
Error: Strings do not match (line 1):
Log file: 'Verifying test files
'.
Test:     'Verifying test files'.
Testing group 'test'
Testing noexamples
Error: Strings do not match (line 1):
Log file: 'Loading package 'noexamples'
'.
Test:     'Loading package 'noexamples''.
Testing noload
Error: Strings do not match (line 1):
Log file: 'Loading package 'noload'
'.
Test:     'Loading package 'noload''.
Testing onerror
Error: Strings do not match (line 1):
Log file: 'Loading configuration file 'config/all.lua'
'.
Test:     'Loading configuration file '../config/all.lua''.
The distance (4) was greater than the maximum (3).
Testing twoerrors
Error: Strings do not match (line 1):
Log file: 'Loading configuration file 'config/all.lua'
'.
Test:     'Loading configuration file '../config/all.lua''.
The distance (4) was greater than the maximum (3).
Checking builds
Checking buildunneclean_0.1.zip
Checking buildafile_0.1.zip
Checking noexamples_0.1.zip
Checking twoerrorsbuild_0.1.zip
Checking buildunnecfiles_0.1.zip
Checking build_0.1.zip
Checking onerrorbuild_0.1.zip
Installing packages
Installing buildunneclean_0.1.zip
Copying package 'buildunneclean'
Package 'buildunneclean' was not installed before
Verifying dependencies
Trying to load package 'buildunneclean'
Installing package 'buildunneclean'
Package 'buildunneclean' successfully installed
Installing buildafile_0.1.zip
Copying package 'buildafile'
Package 'buildafile' was not installed before
Verifying dependencies
Trying to load package 'buildafile'
Installing package 'buildafile'
Package 'buildafile' successfully installed
Installing onerrorbuild_0.1.zip
Copying package 'onerrorbuild'
Package 'onerrorbuild' was not installed before
Verifying dependencies
Trying to load package 'onerrorbuild'
Installing package 'onerrorbuild'
Package 'onerrorbuild' successfully installed
Installing twoerrorsbuild_0.1.zip
Copying package 'twoerrorsbuild'
Package 'twoerrorsbuild' was not installed before
Verifying dependencies
Trying to load package 'twoerrorsbuild'
Installing package 'twoerrorsbuild'
Package 'twoerrorsbuild' successfully installed
Installing noexamples_0.1.zip
Copying package 'noexamples'
Package 'noexamples' was not installed before
Verifying dependencies
Trying to load package 'noexamples'
Installing package 'noexamples'
Package 'noexamples' successfully installed
Installing build_0.1.zip
Copying package 'build'
Package 'build' was not installed before
Verifying dependencies
Trying to load package 'build'
Installing package 'build'
Package 'build' successfully installed
Installing buildunnecfiles_0.1.zip
Copying package 'buildunnecfiles'
Package 'buildunnecfiles' was not installed before
Verifying dependencies
Trying to load package 'buildunnecfiles'
Installing package 'buildunnecfiles'
Package 'buildunnecfiles' successfully installed
Checking observers
Checking chart.tme
Checking map.tme
Checking observer.tme
Checking textscreen.tme
Checking clock.tme
Checking visualtable.tme
Verifying directories
Verifying log
Verifying config
Verifying scripts
Verifying packages

Execution test report:
Tests were executed in 228.0 seconds.
Results were saved in '.terramerun_QTL2D'.
14 out of 136 commands were not successfully executed.
All 7 builds for installed packages were successfully executed.
All 7 builds for local packages were successfully executed.
76 log problems for installed packages were found during the tests.
40 log problems for local packages were found during the tests.
No log file was created.
All observers are saved correctly.
All files and packages are used at least once in the tests.
Summing up, 130 problems were found during the tests.
```

**Comment by pedro-andrade-inpe** [comment]
Implement also execution tests for `forEachNeighborAgent`.
